scania-digital-design-system/tegel
# fix(popover): refactor faulty left-start option for popover positioning

**Describe pull-request**  
Refactored faulty left-start option for popover positioning in both Popover Canvas and Popover Menu.

**Solving issue**  
Fixes: https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-2135

**How to test**  
1. Go to Storybook.
2. Check in Popover Canvas and Popover Menu.
3. Select "Left start" from Canvas position/Menu position control.
4. Make sure the popover opens on the left side of the triggering element, not in the top left corner of the screen.

**Screenshots**  
Expected behavior:
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/b3458a0f-eb11-4d03-a9aa-4c0eb85ed001)
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/b9e77481-61ce-4e31-9612-66ba38f80c4d)


Unwanted behavior:
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/72a11f2b-aafa-4091-a3b3-17eb73d889dc)
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/caddb594-70e9-4ede-ace9-dc47a1e4671a)
